From 41b8ecd6177f3916ec2ad16bf0f1a1551e77bfd4 Mon Sep 17 00:00:00 2001
From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Sat, 26 Feb 2022 15:29:02 +0100
Subject: [PATCH] x86: revert switch to EFI-compatible images

The main difference between the non-EFI and EFI images generated by
OpenWrt is that the former uses an MS-DOS partition table, while the
latter uses GPT. The EFI images still have a BIOS-compatible MBR, so
they work fine on non-EFI systems.
Yet they change the boot partition
---
 targets/x86.inc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/targets/x86.inc b/targets/x86.inc
index f090a9e81e..7aac78684b 100644
--- a/targets/x86.inc
+++ b/targets/x86.inc
@@ -44,8 +44,8 @@ packages {
 config('TARGET_ROOTFS_EXT4FS', false)
 
 defaults {
-	factory = '-squashfs-combined-efi',
+	factory = '-squashfs-combined',
 	factory_ext = {'.img.gz', '.vmdk', '.vdi'},
-	sysupgrade = '-squashfs-combined-efi',
+	sysupgrade = '-squashfs-combined',
 	sysupgrade_ext = '.img.gz',
 }
